hideShowPassword.js

```diff
@@ -52,6 +52,11 @@
     // set after input has been inserted into the DOM.
     enable: canSetInputAttribute,
 
+    // Event to trigger whenever the element is toggled.
+    // For example, if 'focus' it will focus the cursor in the
+    // input element after toggling.
+    triggerOnToggle: '',
```

Comment:
Do you think this should still default to `false`? I know it means a conditional check later, but I don't know what the performance of running a needless `trigger` method is compared to checking for it first.